MediaSystem.cpp

```diff
@@ -210,14 +210,6 @@ class WideVine : public IMediaKeys, public widevine::Cdm::IEventListener
     }
 
     virtual void onKeyStatusesChange(const std::string& session_id) {
-
```

Comment:
Can you eleborate why this complete setion is removed ?

Reply:
@pwielders 
This change is done to ensure that OCDM key_update_callback and keys_updated_callback  are invoked for every license update.
At present, above callbacks are invoked in Widevine callback onKeyStatusesChange().
However Widevine calls onKeyStatusesChange() only if there is change in the keys.
Typically for the given YouTube content, Widevine calls onKeyStatusesChange() only for the first license update.
So for subsequent periodic license updates, OCDM key_update_callback and keys_updated_callback  are not invoked.
This is causing issue in the Cobalt OCDM implementation.